docs/modules/admin/pages/create-voting-decidim.adoc

```diff
@@ -148,7 +148,7 @@ In the email you will receive the instructions on how to decrypt the export file
 
 image::voting-census-export.png[The access codes have been generated - next step: export them.]
 
-The access codes have been generated - next step: export them.
+The access codes have been generated - next step: export them. NB: You can only export one time
```

Comment:
I have just a minor format change, so it's nicer to see on the documentation website. 
 
```suggestion
The access codes have been generated - next step: export them.

NOTE: You can only export access codes one time
```